### ISSUE TITLE
getCompTox() not working anymore because EPA services were retired

### ISSUE THREAD

**Issue by sneumann** [MassBank/RMassBank#310]
Hi,
when running `getCompTox("GWHJZXXIDMPWGX-UHFFFAOYSA-N")` I currently get an error
```
Error in fromJSON(data) : 
  not all data was parsed (0 chars were parsed out of a total of 9093 chars)
```
because https://actorws.epa.gov/index.html?identifier=GWHJZXXIDMPWGX-UHFFFAOYSA-N
tells us:
`ACToR and AcTORWS web services have been retired and are being replaced in the near future with new versions.`

Yours,
Steffen

**Comment by schymane** [comment]
I guess that we should just deprecate this, since there is no concrete definition of `near future` at this stage.

**Comment by tsufz** [comment]
I agree with @schymane to deprecate. This message is life for several weeks (months?) now.
